<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrainLabelingGui</class>
 <widget class="QMainWindow" name="BrainLabelingGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1521</width>
    <height>1113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGraphicsView" name="sagittal_gview"/>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QGraphicsView" name="coronal_gview"/>
        <widget class="QGraphicsView" name="horizontal_gview"/>
        <widget class="QGraphicsView" name="sagittal_tb_gview"/>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="4">
          <widget class="QPushButton" name="button_displayStructures">
           <property name="text">
            <string>Display Structures</string>
           </property>
          </widget>
         </item>
         <item row="4" column="6">
          <widget class="QPushButton" name="button_displayOptions">
           <property name="text">
            <string>Display Options</string>
           </property>
          </widget>
         </item>
         <item row="4" column="5">
          <widget class="QPushButton" name="button_inferSide">
           <property name="text">
            <string>Infer Side</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="lineEdit_username"/>
         </item>
         <item row="4" column="3">
          <widget class="QPushButton" name="button_stop">
           <property name="text">
            <string>Stop Loading</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="7">
          <widget class="QPushButton" name="button_save">
           <property name="text">
            <string>Save Contours</string>
           </property>
          </widget>
         </item>
         <item row="4" column="9">
          <widget class="QPushButton" name="button_load">
           <property name="text">
            <string>Load Contours</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="8">
          <widget class="QPushButton" name="button_saveMarkers">
           <property name="text">
            <string>Save Markers</string>
           </property>
          </widget>
         </item>
         <item row="4" column="10">
          <widget class="QPushButton" name="button_loadMarkers">
           <property name="text">
            <string>Load Markers</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1521</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
